update_rules_content.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Updates the ./tests/content/ok/ directory with the latest rules to test against
+
+function clean_up() {
+    rm -rf "$CLONE_TEMP_DIR"
+}
+trap clean_up EXIT
+
+RULES_REPO="https://gitlab.cee.redhat.com/ccx/ccx-rules-ocp.git"
+CONTENT_DIR="$( cd "$(dirname "$0")" && pwd )/tests/content"
+OK_RULES_DIR="${CONTENT_DIR}/ok"
```

Comment:
Now some tests use `./tests/content/ok` so maybe it's better to use another directory so that it wouldn't break any tests if content changes

Reply:
@Sergey1011010 Isn't that what we want?? For our tests to pass against real data?